applications/source/zeromq-source/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>stream-applications-core</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>zeromq-source</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <name>zeromq-source</name>
+    <description>0MQ source apps</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <zeromq.version>0.5.2</zeromq.version>
```

Comment:
Hm. I wonder why do we need this since we have a `zeromq-supplier` which brings for us SI-zeromq and so on...

Reply:
doesn't seem to resolve that library transitively.